pkg/combustion/registry.go

```diff
@@ -373,6 +376,10 @@ func populateRegistry(ctx *image.Context, images []string) error {
 		if err := generateRegistryTar(ctx, imageTarDest); err != nil {
 			return fmt.Errorf("generating hauler store tar: %w", err)
 		}
+
+		if err := bar.Add(1); err != nil {
+			zap.S().Errorf("Error incrementing the progress bar: %s", err)
```

Comment:
I'm not sure if handling the error like this is the way to go, or if it's even necessary, open to input.

Reply:
We can use the debug level for this, it's not that imporant.